package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>dbow2</name>
+  <version>1.1.0</version>
+  <description>
+    Enhanced hierarchical bag-of-word library for C++
+  </description>
+  <maintainer email="dorian3d@users.noreply.github.com">Dorian Gálvez-López</maintainer>
+  <license>BSD</license>
```

Comment:
Although based in BSD, the DBoW2 license adds an extra clause that allows me to be notified when people use the library. Could you change this to something like `BSD + notification`?

Reply:
I've incorporated the feedback, but you may want to consider using a standard open source licence to better allow for redistribution, as many community packaging channels are now automated and have no means of submitting notice.